components/connectivity-adapter/internal/connector/api.go

```diff
@@ -0,0 +1,71 @@
+package connector
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connector/api"
+	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connector/api/middlewares"
+	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connector/graphql"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	CompassConnectorURL string `envconfig:"default=http://compass-connector.compass-system.svc.cluster.local:3000/graphql"`
+	AdapterBaseURL      string `envconfig:"default=https://adapter-gateway.kyma.local"`
+	AdapterMtlsBaseURL  string `envconfig:"default=https://adapter-gateway-mtls.kyma.local"`
+}
+
+const (
+	timeout = 30 * time.Second
+)
+
+func RegisterHandler(router *mux.Router, config Config) error {
```

Comment:
Consider structuring the code similarly to `appregistry.go` - I think all routes in single place improve readability of the code. But it's just a suggestion.